alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md

```diff
@@ -0,0 +1,121 @@
+# PrometheusScrapeBodySizeLimitHit
+
+## Meaning
+
+The alert `PrometheusScrapeBodySizeLimitHit` is triggered when at least one of
+Prometheus' scrape targets replies with a response body larger than `body_size_limit`.
```

Comment:
Maybe add a sentence explaining that the alert can only fire if someone has configured the body size limit explicitly.